hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/VerificationStrategy.java

```diff
@@ -70,9 +72,11 @@ enum Decision {
      * given this strategy within the given {@link HandleContext}.
      *
      * @param context the context in which this strategy will be used
+     * @param maybeEthSenderKey the key that is the sender of the EVM message, if known
      * @return a predicate that tests whether a given key is a valid signature for a given key
      */
-    default Predicate<Key> asSignatureTestIn(@NonNull final HandleContext context) {
+    default Predicate<Key> asSignatureTestIn(
```

Comment:
Is there an actual unit test anywhere for this interesting and complex method? I see a lot of uses where it is mocked, and one usage (in `HandleSystemContractOperationsTest.dispatchingRespectingGivenStrategy`) where a specific case is tested, and that's it ...